elements/rh-secondary-nav/rh-secondary-nav-menu.css

```diff
@@ -43,14 +43,16 @@
     position: absolute;
     left: 0;
     right: 0;
-    top: var(--_nav-height);
+    /* top: var(--_nav-height); */
```

Comment:
@zeroedin are you good with remove this?

Reply:
Fixed in 007da26350aa49ff05695ad3de41c8f5145a7181